include/triton/Tools/Sys/GetEnv.hpp

```diff
@@ -29,6 +29,7 @@ inline const std::set<std::string> CACHE_INVALIDATING_ENV_VARS = {
     "TRITON_DISABLE_RESHAPE_ENCODING_INFERENCE",
     "TRITON_ENABLE_LLVM_DEBUG",
     "TRITON_LLVM_DEBUG_ONLY",
+    "TRITON_ENABLE_ADDRESS_SANITIZER",
```

Comment:
This flag should be documented in README

Reply:
Updated. Let me know your thoughts on the wording.